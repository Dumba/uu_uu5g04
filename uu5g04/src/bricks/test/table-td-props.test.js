/**
 * Copyright (C) 2019 Unicorn a.s.
 * 
 * This program is free software; you can use it under the terms of the UAF Open License v01 or
 * any later version. The text of the license is available in the file LICENSE or at www.unicorn.com.
 * 
 * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even
 * the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See LICENSE for more details.
 * 
 * You may contact Unicorn a.s. at address: V Kapslovne 2767/2, Praha 3, Czech Republic or
 * at the email: info@unicorn.com.
 */

import React from 'react';
import {shallow} from 'enzyme';
import UU5 from "uu5g04";
import "uu5g04-bricks";
import TestTools from "../../core/test/test-tools.js";
import renderer from 'react-test-renderer';

const TagName = "UU5.Bricks.Table.Td";

const CONFIG = {
  mixins: [
    "UU5.Common.BaseMixin",
    "UU5.Common.ElementaryMixin",
    "UU5.Common.ContentMixin",
    "UU5.Common.ColorSchemaMixin",
    "UU5.Common.NestingLevelMixin",
    "UU5.Common.PureRenderMixin"
  ],
  props: {
    colSpan: {
      values:["3", 3]
    },
    rowSpan: {
      values:[3, "3"]
    }
  },
  requiredProps: {
    parent: shallow(<UU5.Bricks.Table id="parentId"/>).instance()
  },
  opt: {
    shallowOpt: {
      disableLifecycleMethods: false
    },
    enzymeToJson: false
  }
};


describe(`${TagName}`, () => {
  TestTools.testProperties(TagName, CONFIG);
});


describe(`${TagName} docKit examples`, () => {
  it(`${TagName} should render without crash`, () => {
    const wrapper = renderer.create(
      <UU5.Bricks.Container id={"uuID"}>
        <UU5.Bricks.Table id={"uuID2"}>
          {/*@@viewOn:0*/}
          <UU5.Bricks.Table.TBody colorSchema="blue">
            <UU5.Bricks.Table.Tr>
              <UU5.Bricks.Table.Td colSpan={3} rowSpan={"3"} content='Rimmer' />
              <UU5.Bricks.Table.Td colSpan={3} rowSpan={"3"} content='2nd class technician' />
              <UU5.Bricks.Table.Td colSpan={3} rowSpan={"3"} content='comical' />
            </UU5.Bricks.Table.Tr>
            <UU5.Bricks.Table.Tr>
              <UU5.Bricks.Table.Td colSpan={"3"} rowSpan={3} content='Dave Lister' />
              <UU5.Bricks.Table.Td colSpan={"3"} rowSpan={3} content='3rd class technician' />
              <UU5.Bricks.Table.Td colSpan={"3"} rowSpan={3} content='zero' />
            </UU5.Bricks.Table.Tr>
            <UU5.Bricks.Table.Tr>
              <UU5.Bricks.Table.Td content='Kristine Kochanski' />
              <UU5.Bricks.Table.Td content='1st console officer' />
              <UU5.Bricks.Table.Td content='high' />
            </UU5.Bricks.Table.Tr>
          </UU5.Bricks.Table.TBody>
          {/*@@viewOff:0*/}
        </UU5.Bricks.Table>
      </UU5.Bricks.Container>
    ).toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});










