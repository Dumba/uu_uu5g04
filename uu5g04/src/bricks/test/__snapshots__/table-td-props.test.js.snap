// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UU5.Bricks.Table.Td checkGrammar 1`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td checkGrammar 2`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td checkGrammar 3`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td checkGrammar 4`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td checkHighlight 1`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td checkHighlight 2`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td checkHighlight 3`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td checkHighlight 4`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td checkSpaces 1`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td checkSpaces 2`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td checkSpaces 3`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td checkSpaces 4`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td className 1`] = `
<td
  className="uu5-bricks-table-td uu5-common-text my-classname"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td colSpan 1`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  colSpan="3"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td colSpan 2`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  colSpan={3}
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td colSpan 3`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  colSpan={3}
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td colSpan 4`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  colSpan="3"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td colorSchema 1`] = `
<td
  className="color-schema-default uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td colorSchema 2`] = `
<td
  className="color-schema-red uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td colorSchema 3`] = `
<td
  className="color-schema-red uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td colorSchema 4`] = `
<td
  className="color-schema-red-rich uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td colorSchema 5`] = `
<td
  className="color-schema-red-rich uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td colorSchema 6`] = `
<td
  className="color-schema-green uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td colorSchema 7`] = `
<td
  className="color-schema-green uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td colorSchema 8`] = `
<td
  className="color-schema-green-rich uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td colorSchema 9`] = `
<td
  className="color-schema-green-rich uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td colorSchema 10`] = `
<td
  className="color-schema-yellow uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td colorSchema 11`] = `
<td
  className="color-schema-yellow uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td colorSchema 12`] = `
<td
  className="color-schema-yellow-rich uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td colorSchema 13`] = `
<td
  className="color-schema-yellow-rich uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td colorSchema 14`] = `
<td
  className="color-schema-purple uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td colorSchema 15`] = `
<td
  className="color-schema-purple uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td colorSchema 16`] = `
<td
  className="color-schema-purple-rich uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td colorSchema 17`] = `
<td
  className="color-schema-purple-rich uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td colorSchema 18`] = `
<td
  className="color-schema-brown uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td colorSchema 19`] = `
<td
  className="color-schema-brown uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td colorSchema 20`] = `
<td
  className="color-schema-brown-rich uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td colorSchema 21`] = `
<td
  className="color-schema-brown-rich uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td colorSchema 22`] = `
<td
  className="color-schema-orange uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td colorSchema 23`] = `
<td
  className="color-schema-orange uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td colorSchema 24`] = `
<td
  className="color-schema-orange-rich uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td colorSchema 25`] = `
<td
  className="color-schema-orange-rich uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td colorSchema 26`] = `
<td
  className="color-schema-cyan uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td colorSchema 27`] = `
<td
  className="color-schema-cyan uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td colorSchema 28`] = `
<td
  className="color-schema-cyan-rich uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td colorSchema 29`] = `
<td
  className="color-schema-cyan-rich uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td colorSchema 30`] = `
<td
  className="color-schema-grey uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td colorSchema 31`] = `
<td
  className="color-schema-grey uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td colorSchema 32`] = `
<td
  className="color-schema-grey-rich uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td colorSchema 33`] = `
<td
  className="color-schema-grey-rich uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td colorSchema 34`] = `
<td
  className="color-schema-pink uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td colorSchema 35`] = `
<td
  className="color-schema-pink uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td colorSchema 36`] = `
<td
  className="color-schema-pink-rich uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td colorSchema 37`] = `
<td
  className="color-schema-pink-rich uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td colorSchema 38`] = `
<td
  className="color-schema-pink uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td content 1`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
>
  12345
</td>
`;

exports[`UU5.Bricks.Table.Td content 2`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
>
  <UU5.Common.TextCorrectorContextConsumer
    key="0"
    text="This is Content from ContentMixin"
  />
</td>
`;

exports[`UU5.Bricks.Table.Td content 3`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
>
  <UU5.Common.TextCorrectorContextConsumer
    key="0"
    text="This is Content from ContentMixin"
  />
</td>
`;

exports[`UU5.Bricks.Table.Td content 4`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
>
  12345
</td>
`;

exports[`UU5.Bricks.Table.Td controlled 1`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td controlled 2`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td controlled 3`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td controlled 4`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td default props 1`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td disabled 1`] = `
<td
  className="uu5-bricks-table-td uu5-common-text uu5-common-disabled"
  id="uuID"
  style={Object {}}
  title={null}
>
  <span
    className="uu5-common-disabled-cover"
    key="disabledCover"
  />
</td>
`;

exports[`UU5.Bricks.Table.Td disabled 2`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td disabled 3`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td disabled 4`] = `
<td
  className="uu5-bricks-table-td uu5-common-text uu5-common-disabled"
  id="uuID"
  style={Object {}}
  title={null}
>
  <span
    className="uu5-common-disabled-cover"
    key="disabledCover"
  />
</td>
`;

exports[`UU5.Bricks.Table.Td docKit examples UU5.Bricks.Table.Td should render without crash 1`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
>
  <div
    className="uu5-bricks-table"
    id="uuID2"
    style={Object {}}
    title={null}
  >
    <table
      className="uu5-bricks-table-table"
    >
      <tbody
        className="color-schema-blue uu5-bricks-table-tbody"
        style={Object {}}
        title={null}
      >
        <tr
          className="uu5-bricks-table-tr"
          style={Object {}}
          title={null}
        >
          <td
            className="uu5-bricks-table-td uu5-common-text"
            colSpan={3}
            rowSpan="3"
            style={Object {}}
            title={null}
          >
            Rimmer
          </td>
          <td
            className="uu5-bricks-table-td uu5-common-text"
            colSpan={3}
            rowSpan="3"
            style={Object {}}
            title={null}
          >
            2nd class technician
          </td>
          <td
            className="uu5-bricks-table-td uu5-common-text"
            colSpan={3}
            rowSpan="3"
            style={Object {}}
            title={null}
          >
            comical
          </td>
        </tr>
        <tr
          className="uu5-bricks-table-tr"
          style={Object {}}
          title={null}
        >
          <td
            className="uu5-bricks-table-td uu5-common-text"
            colSpan="3"
            rowSpan={3}
            style={Object {}}
            title={null}
          >
            Dave Lister
          </td>
          <td
            className="uu5-bricks-table-td uu5-common-text"
            colSpan="3"
            rowSpan={3}
            style={Object {}}
            title={null}
          >
            3rd class technician
          </td>
          <td
            className="uu5-bricks-table-td uu5-common-text"
            colSpan="3"
            rowSpan={3}
            style={Object {}}
            title={null}
          >
            zero
          </td>
        </tr>
        <tr
          className="uu5-bricks-table-tr"
          style={Object {}}
          title={null}
        >
          <td
            className="uu5-bricks-table-td uu5-common-text"
            style={Object {}}
            title={null}
          >
            Kristine Kochanski
          </td>
          <td
            className="uu5-bricks-table-td uu5-common-text"
            style={Object {}}
            title={null}
          >
            1st console officer
          </td>
          <td
            className="uu5-bricks-table-td uu5-common-text"
            style={Object {}}
            title={null}
          >
            high
          </td>
        </tr>
      </tbody>
    </table>
  </div>
</div>
`;

exports[`UU5.Bricks.Table.Td dynamic 1`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td dynamic 2`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td dynamic 3`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td dynamic 4`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td hidden 1`] = `
<td
  className="uu5-bricks-table-td uu5-common-text uu5-common-hidden"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td hidden 2`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td hidden 3`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td hidden 4`] = `
<td
  className="uu5-bricks-table-td uu5-common-text uu5-common-hidden"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td id 1`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="myId"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td ignoreHTML 1`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td ignoreHTML 2`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td ignoreHTML 3`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td ignoreHTML 4`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td mainAttrs 1`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td name 1`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  name="myName"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td nestingLevel 1`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td nestingLevel 2`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td nestingLevel 3`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td nestingLevel 4`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td nestingLevel 5`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td nestingLevel 6`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td nestingLevel 7`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td nestingLevel 8`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td nestingLevel 9`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td nestingLevel 10`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td nestingLevel 11`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td nestingLevel 12`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td nestingLevel 13`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td nestingLevel 14`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td nestingLevel 15`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td nestingLevel 16`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td nestingLevel 17`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td nestingLevel 18`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td nestingLevel 19`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td nestingLevel 20`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td noIndex 1`] = `
<td
  className="uu5-bricks-table-td uu5-common-text uu5-noindex"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td noIndex 2`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td noIndex 3`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td noIndex 4`] = `
<td
  className="uu5-bricks-table-td uu5-common-text uu5-noindex"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td pureRender 1`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td pureRender 2`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td pureRender 3`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td pureRender 4`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td rowSpan 1`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  rowSpan={3}
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td rowSpan 2`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  rowSpan="3"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td rowSpan 3`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  rowSpan="3"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td rowSpan 4`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  rowSpan={3}
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td selected 1`] = `
<td
  className="uu5-bricks-table-td uu5-common-text uu5-common-selected"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td selected 2`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td selected 3`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td selected 4`] = `
<td
  className="uu5-bricks-table-td uu5-common-text uu5-common-selected"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td style 1`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={
    Object {
      "borderRadius": "2px",
      "padding": "8px",
    }
  }
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td style 2`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={
    Object {
      "backgroundColor": "red",
      "color": "blue",
    }
  }
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td style 3`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={
    Object {
      "backgroundColor": "red",
      "color": "blue",
    }
  }
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td style 4`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={
    Object {
      "borderRadius": "2px",
      "padding": "8px",
    }
  }
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td textCorrector 1`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td textCorrector 2`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td textCorrector 3`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td textCorrector 4`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Td tooltip 1`] = `
<td
  className="uu5-bricks-table-td uu5-common-text"
  id="uuID"
  style={Object {}}
  title="My Tooltip"
/>
`;
